users/models.py

```diff
@@ -74,9 +100,34 @@ def has_subjudge_auth(self):
                     or (self.user_level == SUB_JUDGE))
         return has_auth
 
+    def get_full_name(self):
+        return self.username
+
+    def get_short_name(self):
+        return self.username
+
+    def has_perm(self, perm, obj=None):
+        # Simplest possible answer: Yes, always (To be constructed later)
+        return True
+
+    def has_module_perms(self, app_label):
+        # Simplest possible answer: Yes, always (To be constructed later)
+        return True
+
     def __unicode__(self):
         return self.username
 
+    def __str__(self):  # __unicode__ on Python 2
```

Comment:
This is for python 3. Python 2 version is __unicode__, which is write in line 117~118.